lib/util/search.js

```diff
@@ -49,8 +50,16 @@ function SearchClient(crowi, esUri) {
   })
 
   this.registerUpdateEvent()
+}
 
-  this.mappingFile = crowi.resourceDir + 'search/mappings.json'
+SearchClient.prototype.selectMappingFile = function() {
```

Comment:
This function is depending on the result of `checkESVersion()` that is called in the bootstrap process of `lib/crowi/index.js`.
Is it needed to check wether the plugin information loaded correctly or not?

Reply:
`selectMappingFile()` return 'mappings.json' in default.
So it may not be necessary to know whether `checkESVersion()` ran in the past or not.
I think run `checkESVersion()` before `selectMappingFile()` is more kind (can get the latest information).